packages/qwik/src/qwikloader.ts

```diff
@@ -36,13 +44,13 @@ export const qwikLoader = (doc: Document, hasInitialized?: number) => {
     );
   };
 
-  const resolveContainer = (containerEl: Element) => {
-    if ((containerEl as QContainerElement)[Q_JSON] === undefined) {
+  const resolveContainer = (containerEl: Element & QContainerElement) => {
```

Comment:
Can't we make QContainerElement be also Element?

Reply:
yup we should